source/jsonnet/england-wales/household/blocks/who-lives-here/primary_person_list_collector.jsonnet

```diff
@@ -48,13 +48,18 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
     },
     definitions: [
       {
-        title: 'What does “usually live” mean?',
+        title: 'What we mean by “usually live”?',
         contents: [
           {
-            description: 'It means the address at which you generally spend most time. For most people, this will be their permanent or family home.',
+            description: {
+              text: 'This is often your permanent or family home.<br><br>If you have more than one address, include yourself at the home address where you spend most of your time. If you split your time equally then use the home address where you are staying overnight on {census_date}.',
```

Comment:
`<br>` tags shouldn't be used. This should be done with two description elements.

Reply:
Made it separate description, thanks 👍